packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -805,7 +826,11 @@ async function runScript() {
         case 'String': return String;
         default: return undefined;
       }
-    }).alias({
+    })
+    .string(['inputTemplate', 'outputTemplate'])
```

Comment:
Please move this into the existing `string` call earlier, and group the new `default` calls with the others.